sbag/generators/javascript/entities/modules/__entity_name__/__entity_name__-list/__entity_name__-list.component.html.jinja

```diff
@@ -52,4 +52,20 @@
 </div>
 <div *ngIf="{{entity.name|plural|lower}}?.length == 0" class="alert alert-warning" role="alert" style="text-align:center;">
   There are no {{ entity_name_plural }} added!
-</div>
\ No newline at end of file
+</div>
+
+<ng-template #mymodal let-modal>
+  <div class="modal-header">
+    <h4 class="modal-title" id="modal-basic-title">Delete</h4>
+    <button type="button" class="close" aria-label="Close" (click)="modal.dismiss('Cross click')">
+      <span aria-hidden="true">&times;</span>
+    </button>
+  </div>
+  <div class="modal-body">
+    <h5 class="text-danger">Are you sure that you want to delete {{entity_name}}?</h5>
+  </div>
+  <div class="modal-footer">
+    <button type="button" class="btn btn-danger" (click)="delete{{entity.name}}(idToDelete);modal.dismiss('Cross click')">Delete</button>
+    <button type="button" class="btn btn-secondary" (click)="modal.dismiss('Cross click')">Cancel</button>
+  </div>
+</ng-template>
```

Comment:
Can this be a separate component so we can use it here and in entity-preview delete? Both components have similar code, that's why I'm asking.